### ISSUE TITLE
Module BrowserSync is not available! You either misspelled the module name [...]

### ISSUE THREAD

**Issue by xdivby0** [BrowserSync/browser-sync#1885]
### Issue details

I am trying to use browser-sync inside a ghost template so I can reload the browser when changes have been made. When I navigate to localhost:3001, I can see the UI skeleton without any data (still in loading state):
![grafik](https://user-images.githubusercontent.com/53540163/124668241-da1ee400-deb0-11eb-95e1-e5e81655d7c8.png)

When I look into the console (of the browser-sync UI at port 3001), I can see these errors, what did I do wrong?
![grafik](https://user-images.githubusercontent.com/53540163/124668036-94fab200-deb0-11eb-8efb-7b10fbd30784.png)
I'll attach the errors in text form at the end of this issue.

### Steps to reproduce/test case

I am simply using 
```
browserSync.init({
  proxy: "localhost:2368",
});
```
in my gulp file and I tried browser-sync versions 2.27.4, 2.27.1 and 2.26 (with no success).

### Please specify which version of Browsersync, node and npm you're running

- Browsersync [2.27.4, 2.27.1, 2.26]
- Node        [12.22.1]
- Npm         [6.14.12]

### Affected platforms
- [x] windows

### Browsersync use-case
- [X] Gulp

```
const browserSync = require("browser-sync").create();
browserSync.init({
  proxy: "localhost:2368",
});
```
and then `browserSync.stream()` at some places but this shouldn't be related to the problem at hand.

Angular error in textform:
```
Uncaught TypeError: e.lowercase is not a function
    Angular 3
    Webpack 5
angular-sanitize.js:265:32
Uncaught Error: [$injector:nomod] Module 'BrowserSync' is not available! You either misspelled the module name or forgot to load it. If registering a module ensure that you specify the dependencies as the second argument.
https://errors.angularjs.org/1.8.2/$injector/nomod?p0=BrowserSync
    Angular 4
    <anonymous> pages-config.js:6
    <anonymous> pages-config.js:134
angular.js:138:11
Uncaught Error: [$injector:nomod] Module 'BrowserSync' is not available! You either misspelled the module name or forgot to load it. If registering a module ensure that you specify the dependencies as the second argument.
https://errors.angularjs.org/1.8.2/$injector/nomod?p0=BrowserSync
    Angular 4
    <anonymous> client-js.js:6
    <anonymous> client-js.js:94
angular.js:138:11
Uncaught Error: [$injector:modulerr] Failed to instantiate module BrowserSync due to:
[$injector:nomod] Module 'BrowserSync' is not available! You either misspelled the module name or forgot to load it. If registering a module ensure that you specify the dependencies as the second argument.

https://errors.angularjs.org/1.8.2/$injector/modulerr?p0=Bro…0http%3A%2F%2Flocalhost%3A3001%2Fjs%2Fapp.js%3A2%3A272529%0A
    Angular 15
    Webpack 5
```

**Comment by itergator** [comment]
I am getting exactly the same issue with the following config:



**Affected Platform**
Mac

node 14.15.4
npm 6.14.10

**Affected Framework**
django 2.2.16

**Browsersync use-case**
grunt v. 1.2.1
grunt-contrib-watch v. 1.1.0
grunt-browser-sync v. 2.2.0
browser-sync v. 2.27.4

```
browserSync: {
  dev: {
      bsFiles: {
          src : '*/templates/**/*.html'
      },
      options: {
          proxy: "127.0.0.1:8000",
          watch: true
      }
  }
},
```

following install instructions from the grunt docs

**Comment by schonhoff** [comment]
Hello, I have the same issue. 
It is because of this:
https://stackoverflow.com/questions/50448326/uncaught-typeerror-angular-lowercase-is-not-a-function
The angular version was updated for 6 days.

**Comment by goraxan** [comment]
I have the same problem, any idea about when this will be fixed?

**Comment by etlam** [comment]
Same problem here using Node 14 LTS on linux.

**Comment by wingcommander2020** [comment]
I am getting the same error

Node v14.17.3
Linux Mint 20. 

├─┬ browser-sync@2.27.4
│ ├─┬ browser-sync-client@2.27.4
│ ├─┬ browser-sync-ui@2.27.4

On Windows with same Node version it seems to work though.

![MicrosoftTeams-image (1)](https://user-images.githubusercontent.com/60811299/125921306-aefe532b-ec56-4775-b023-13d127f7f6d1.png)

**Comment by doptrois** [comment]
I needed to downgrade to 2.24.7.
I'm using Node.js 15.14.0 and npm 7.7.6 (in case it's relevant to you).

**Comment by shakyShane** [comment]
Fixed in `browser-sync@2.27.5` - thank you for reporting :)

**Comment by MattrCoUk** [comment]
I'm getting same issue in both @2.27.5 and @2.27.7
Win10,  npm 8.1.2
trying command line `browser-sync start --proxy...`
is there a workaround?

**Comment by karneaud** [comment]
+1 @MattrCoUk

**Comment by jakewhiteley** [comment]
I am getting this issue also on windows 10.

I'm using browser-sync `2.27.7`, and node `16.13.0`

**Comment by nurtext** [comment]
Still defect for me on  browser-sync 2.27.7 - am I missing something? Thought this has been fixed @shakyShane ?

**Comment by dodaydream** [comment]
I am also getting this issue on browser-sync `2.27.7` and node `17.2.0`, browser-sync-webpack-plugin `2.3.0`

I'm using browser-sync via laravel-mix `6.0.25`

**Comment by ronilaukkarinen** [comment]
I use settings very rarely so didn't notice this before. I'm experiencing this right now in version 2.27.7.

**Comment by ronilaukkarinen** [comment]
Only workaround for now:

``` bash
npm install browser-sync@2.23.6 --save-dev
```

**Comment by gardelin** [comment]
I'm also getting exactly the same issue with the same error messages in the console.

```
Mac OS
node 14.16
npm 6.14.12
browser-sync 2.27.7
laravel-mix 6.0.41
```

Like @ronilaukkarinen mentioned it only worked downgrading browser-sync to `2.23.6`

**Comment by trasherdk** [comment]
Downgrading to `browser-sync 2.23.6` results in `39 vulnerabilities (6 low, 7 moderate, 16 high, 10 critical)` :sob: 

Gonna put `browser-sync` on ice until this is fixed.

**Comment by davide93** [comment]
I'm getting same issue, v. 2.27.7 - node v. 16.14.0

Any updates?
Ty

**Comment by trasherdk** [comment]
Okay. By now, this is a known issue, so spamming 252 accounts with `me too` is not that helpful.
I'd much rather see a post, letting me know that the problem is solved, including those `vulnerabilities`

**Comment by davide93** [comment]
> Okay. By now, this is a known issue, so spamming 252 accounts with `me too` is not that helpful. I'd much rather see a post, letting me know that the problem is solved, including those `vulnerabilities`

It's a know issue but still closed. So, "me too" with envirioments details it's useful to me.

P.s. You can to turn off subscription from this issue if you don't want to receive more updates.

**Comment by lokesh1729** [comment]
To whoever is facing this issue. I cracked the problem.

1. @shakyShane had upgraded the angular deps in 2.27.5 and released it - https://github.com/BrowserSync/browser-sync/compare/v2.27.4...v2.27.5 - but he did not merge them to master - don't know why
2. They were merged to master in v2.27.8 - https://github.com/BrowserSync/browser-sync/pull/1940 & https://github.com/BrowserSync/browser-sync/compare/v2.27.4...v2.27.8
3. So, the versions >=v2.27.5 will not have this issue except v2.27.6 & v2.27.7 because the angular upgrade changes were not present.

So, the solution is simply do `npm install browser-sync@^2.27.8` - it will fix the problem

**Comment by karneaud** [comment]
> To whoever is facing this issue. I cracked the problem.
> 
> 1. @shakyShane had upgraded the angular deps in 2.27.5 and released it - [v2.27.4...v2.27.5](https://github.com/BrowserSync/browser-sync/compare/v2.27.4...v2.27.5) - but he did not merge them to master - don't know why
> 2. They were merged to master in v2.27.8 - [more version bumps + github actions #1940](https://github.com/BrowserSync/browser-sync/pull/1940) & [v2.27.4...v2.27.8](https://github.com/BrowserSync/browser-sync/compare/v2.27.4...v2.27.8)
> 3. So, this issue is fixed in all versions >=v2.27.5  except v2.27.6 & v2.27.7 because the angular upgrade changes were not present. I don't know why the angular upgrade changes are not carry forwarded to v2.27.6 ([v2.27.4...v2.27.6](https://github.com/BrowserSync/browser-sync/compare/v2.27.4...v2.27.6)) & v2.27.7 ([v2.27.4...v2.27.7](https://github.com/BrowserSync/browser-sync/compare/v2.27.4...v2.27.7)) @shakyShane
> 
> So, the solution is simply do `npm install browser-sync@^2.27.8` - it will fix the problem

Just tried this running yarn 1.22.* and node v16.19 and it was unsuccessful running in an Android 8.0 Chrome browser

**Comment by lokesh1729** [comment]
@karneaud - what is `yarn 1.22.*` ? you should install version greater than or equal to `2.27.8`. Do `yarn install browser-sync@^2.27.8`
